database-attachments.md

```diff
@@ -166,3 +167,32 @@ Example:
     $user = $file->attachment;
     
 For more information read the [polymorphic relationships](../database/relations#polymorphic-relations)
+
+<a name="attachments-validation-example"></a>
+### Validation example
+
+This section shows how you can validate `attachMany` attachments.
+
+~~~php
```

Comment:
Code examples aren't enclosed with `~~~php`, they just need to be indented by four spaces